packages/platform/types/event-types/event-types_2024_06_14/inputs/booking-fields.input.ts

```diff
@@ -58,6 +58,11 @@ export class EmailFieldInput_2024_06_14 {
   @DocsProperty()
   label?: string;
 
+  @IsBoolean()
+  @IsOptional()
+  @DocsProperty()
+  required = true;
```

Comment:
Why does it have to be required if we can have phone based bookings aka without email?

Reply:
In the request payload of the PR description there is only attendee phone number.

Reply:
there was no required field defined for email i just set the default value true if it hasn't been passed as false or empty